guac_core/src/crypto.rs

```diff
@@ -38,11 +104,21 @@ impl Crypto {
                 .parse()
                 .unwrap(),
             balance: 1_000_000_000_000u64.into(),
+            // TODO: Proper connecting
```

Comment:
we may have to dip into Rita for this, right now we have a list of url's for full nodes that you can just pull from bu the question is if we want to change the settings names to be more general than the dao stuff.